src/closure.js

```diff
@@ -1,13 +1,20 @@
 // Complete the following functions.
 
 const counter = () => {
+  let count = 0;
+  return () => ++count;
```

Comment:
So here you're using `++` as a prefix operator which increments the value BEFORE returning it.  Kind of a funny difference that trips people up from time to time.